clang-tools-extra/clang-tidy/add_new_check.py

```diff
@@ -324,9 +329,14 @@ def add_release_notes(module_path, module, check_name, description):
 
 
 # Adds a test for the check.
-def write_test(module_path, module, check_name, test_extension, test_standard):
-    if test_standard:
-        test_standard = f"-std={test_standard}-or-later "
+def write_test(
+    module_path: str,
+    module: str,
+    check_name: str,
+    test_extension: str,
+    test_standard: Optional[str],
+) -> None:
+    test_standard = f"-std={test_standard}-or-later " if test_standard else ""
```

Comment:
This is the minor bug fix. If `test_standard` is `None`, it is wrongly formatted into the string a few lines down.